From: Donet Tom <donettom@linux.ibm.com>
Subject: drivers/base/node: restore removed extra line
Date: Mon, 9 Jun 2025 09:03:54 -0500

Re-added the extra blank line at the end of the memory.h that was removed
by the commit 'commit d61861e3bb35 ("drivers/base/node: optimize memory
block registration to reduce boot time")'

Link: https://lkml.kernel.org/r/20250609140354.467908-1-donettom@linux.ibm.com
Signed-off-by: Donet Tom <donettom@linux.ibm.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/memory.h |    1 +
 1 file changed, 1 insertion(+)

--- a/include/linux/memory.h~drivers-base-node-optimize-memory-block-registration-to-reduce-boot-time-fix
+++ a/include/linux/memory.h
@@ -216,4 +216,5 @@ unsigned long memory_block_advised_max_s
  * can sleep.
  */
 extern struct mutex text_mutex;
+
 #endif /* _LINUX_MEMORY_H_ */
_
